openebs/openebs-provisioner.go

```diff
@@ -240,3 +249,12 @@ func main() {
 	}
 
 }
+
+// GetPersistentVolumeClass returns StorageClassName.
+func GetStorageClassName(options controller.VolumeOptions) *string {
+	// Use beta annotation first
+	if class, found := options.PVC.Annotations[BetaStorageClassAnnotation]; found {
```

Comment:
Need to do a nil check of options.PVC

Reply:
That is already handled by k8s-provisioner-controller, if there is No `storageclass` then controller will error out with this log.
`Error getting claim "default/demo-vol1-claim"'s StorageClass's fields: StorageClass "" not found`